helpdesk_mgmt_activity/models/mail_activity.py

```diff
@@ -0,0 +1,19 @@
+from odoo import fields, models
+
+
+class MailActivity(models.Model):
+    _inherit = "mail.activity"
+
+    ticket_id = fields.Many2one(
+        comodel_name="helpdesk.ticket",
+        help="Activity created from helpdesk ticket"
+        "After closing this activity, ticket is moved to done stage",
+    )
+
+    def _action_done(self, feedback=False, attachment_ids=None):
+        # Get closed stage for ticket
+        for ticket in self.mapped("ticket_id"):
```

Comment:
Odoo 16.0 support mapping recordsets like this: `for ticket in self.ticket_id:`

if self is multi record set, then the `self.ticket_id` should return records: `helpdesk.ticket(1, 2, 3)`